CoreFoundation/CMakeLists.txt

```diff
@@ -512,10 +525,13 @@ if(CMAKE_SYSTEM_NAME STREQUAL Darwin)
                           -Xlinker;-alias_list;-Xlinker;Base.subproj/DarwinSymbolAliases;-twolevel_namespace;-sectcreate;__UNICODE;__csbitmaps;CharacterSets/CFCharacterSetBitmaps.bitmap;-sectcreate;__UNICODE;__properties;CharacterSets/CFUniCharPropertyDatabase.data;-sectcreate;__UNICODE;__data;CharacterSets/CFUnicodeData-L.mapping;-segprot;__UNICODE;r;r)
 endif()
 
+if(CF_DEPLOYMENT_SWIFT)
+  set(CF_SWIFT_TARGETS CFXMLInterface)
+endif()
 install(TARGETS
           CoreFoundation
           CFURLSessionInterface
-          CFXMLInterface
+          ${CF_SWIFT_TARGETS}
```

Comment:
I think it may be nicer to just use a property that you append to, rather than doing the variable expansion here.

Reply:
I may be misunderstanding something. Do you mean use set_property() for something? If so, I'm not quite sure how you mean to apply it here.

For the moment, I've pulled out CFXMLInterface into a separate install(), which avoids the variable expansion. Maybe this is better too?

Reply:
Im basically trying to avoid the whole variable soup that CMake <3.0 used to suffer from.  Adding a second `install(TARGETS)` in an `if` is another approach.